### PR TITLE
Volume password

### PR DESCRIPTION
**Pull request by edevenport**
### What does this PR do?

Allows a image root password to be set for server console access. SSH keys are required and now an password is optional.

### Tests written?

No

### Commits signed with GPG?

No

### CODE REVIEW

File: doc/topics/cloud/profitbricks.rst [hunk 1]
```diff
@@ -185,6 +186,13 @@ disk_type
     This option allow the disk type to be set to HDD or SSD. The default is
     HDD.
 
+image_password
+    A password is set on the image for the "root" or "Administrator" account.
+    This field may only be set during volume creation. Only valid with 
+    ProfitBricks supplied HDD (not ISO) images. The password must contain at 
+    least 8 and no more than 50 characters. Only these characters are 
+    allowed: [a-z][A-Z][0-9]
+
```
Comment (rallytime) [inline doc/topics/cloud/profitbricks.rst]: Can you add a `.. versionadded:: Fluorine` tag to this new option?

**Review by rallytime — CHANGES_REQUESTED** [review]
I have one small request, otherwise LGTM.

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
@edevenport Ah, there's a small lint error: https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/20450/violations/file/salt/cloud/clouds/profitbricks.py/

**Comment by denza** [comment]
@rallytime Thanks for heads up. Fixed

**Comment by rallytime** [comment]
re-run py

**Pull request merged**